core/src/main/java/com/graphhopper/routing/util/parsers/helpers/OSMValueExtractor.java

```diff
@@ -80,6 +86,12 @@ public static double stringToMeter(String value) {
         value = FEET_PATTERN.matcher(value).replaceAll("ft");
         if (isInvalidValue(value))
             return Double.NaN;
+        
+        // only used for maxheight
+        if ("below_default".equals(value)) {
+            return 2; // assume that cars are able to pass
+        }
```

Comment:
I would not do this and instead ignore all these soft tags. IMO tagging a height with such a value is a tagging issue and we should not use it or even convert it into an explicit max height.

Reply:
`below_default` is an accepted and documented value. Projects like [StreetComplete](https://github.com/westnordost/StreetComplete) make use of it: https://github.com/westnordost/StreetComplete/blob/ed6ab1426f5c26dc4779c2c84118483a4a84a6eb/app/src/main/java/de/westnordost/streetcomplete/quests/max_height/AddMaxHeight.kt#L68-L77

Reply:
Ok, yes. I can see it in the wiki. It is not really useful IMO and seems to be mostly used by German mappers. My problem with this approach is for GraphHopper itself: if we would do this then some limits have not the meaning they have in real world. So I would keep this for another issue.